### DISCUSSION TITLE
When I add a new UserControl Why x:Class not match the real program path?

### DISCUSSION THREAD

**Discussion by TanatipUTL3** [AvaloniaUI/AvaloniaVS#474] [Q&A]
{Note: I'm a newbie for Avalonia and XAML lol}

I found an issue creating UserControl (See image below) Why x:Class="AvaloniaApplication1.UserControl1"? I think it should be x:Class="AvaloniaApplication1.Views.UserControl1"?? 

> ENV.
Visual Studio Community 2022 [V. 17.9.6]
Avalonia for Visual Studio 2022 [V. 11.5]
Avalonia Template Studio [V. 1.3]

![Attach1](https://github.com/AvaloniaUI/AvaloniaVS/assets/108796329/71fa5329-1bdc-42ea-bc7e-ce1197c030ed)
![main](https://github.com/AvaloniaUI/AvaloniaVS/assets/108796329/def1dc08-8257-4d66-8072-71830d231ccb)

**Comment by maxkatz6** [comment]
It's a namespace + class name. So, if your namespace is different from the physical path, it will be different in x:Class as well.

**Reply by TanatipUTL3** [reply]
🙏Thank you for your reply! Do you mean my physical path is different from my program namespace? How can I fix this problem to correct the path?
![Attach2](https://github.com/AvaloniaUI/AvaloniaVS/assets/108796329/8a871bca-5b46-4dfb-b958-9e6b74fd75c4)
![Attach3](https://github.com/AvaloniaUI/AvaloniaVS/assets/108796329/bb260f54-aada-4981-aa4e-429cecf1941c)

**Reply by maxkatz6** [reply]
Edit your C# class namespace in UserControl1.axaml.cs.

**Reply by TanatipUTL3** [reply]
Is working!! Thank you very much! 
![Attach4](https://github.com/AvaloniaUI/AvaloniaVS/assets/108796329/8cdb4da2-6424-4314-8e30-7fbd2efb6e03)
